pkg/vgmanager/filter_test.go

```diff
@@ -0,0 +1,137 @@
+package vgmanager
+
+import (
+	"testing"
+
+	"github.com/red-hat-storage/lvm-operator/pkg/internal"
+	"github.com/stretchr/testify/assert"
+)
+
+type filterTestCase struct {
+	label     string
+	device    internal.BlockDevice
+	expected  bool
+	expectErr bool
+}
+
+func TestNotReadOnly(t *testing.T) {
+	matcher := notReadOnly
```

Comment:
do we really need this matcher keyword? instead, we can directly make use of the constants.

Reply:
Done.